src/Joomlatools/Console/Command/Versions.php

```diff
@@ -90,19 +94,32 @@ protected function execute(InputInterface $input, OutputInterface $output)
     public function setRepository($repository)
     {
         if ($repository == 'platform') {
-            $repository = 'git@github.com:joomlatools/joomlatools-platform.git';
+            $repository = Versions::REPO_JOOMLATOOLS_PLATFORM;
         }
 
         $this->repository = $repository;
 
-        self::$file = realpath(__DIR__.'/../../../../bin/.files/cache').'/' . md5($this->repository) . '/.versions';
+        self::$file = Util::getWritablePath() . '/cache/' . md5($this->repository) . '/.versions';
     }
 
     public function getRepository()
     {
         return $this->repository;
     }
 
+    /**
+     * Check if the repository is a valid Git repository.
+     *
+     * @return bool
+     */
+    public function isGitRepository()
```

Comment:
We've added this function to determine whether the URL passed in as the `--repo` option is a valid Git repo or not:

* if it is, we clone it,
* it not, we download it.

This makes it work automatically; if you pass a URL pointing to a gzipped tar archive, it will download and install that.